cross-project-tests/debuginfo-tests/dexter/dex/tools/__init__.py

```diff
@@ -5,6 +5,32 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-from dex.tools.Main import Context, get_tool_names, get_tools_directory, main, tool_main
+from dex.tools.Main import Context, main, tool_main
 from dex.tools.TestToolBase import TestToolBase
 from dex.tools.ToolBase import ToolBase
+
+def get_tool_names():
+    """Returns a list of expected DExTer Tools"""
+    return ["help", "list-debuggers", "no-tool-", "run-debugger-internal-", "test", "view"]
+
+def get_tools():
+    """Returns a dictionary of expected DExTer Tools"""
+    return _the_tools
+
+
+from .help import Tool as help_tool
+from .list_debuggers import Tool as list_debuggers_tool
+from .no_tool_ import Tool as no_tool_tool
+from .run_debugger_internal_ import Tool as run_debugger_internal_tool
+from .test import Tool as test_tool
+from .view import Tool as view_tool
+
+_the_tools = {
+      "help" : help_tool,
+      "list-debuggers" : list_debuggers_tool,
+      "no_tool_" : no_tool_tool,
```

Comment:
nit: `no-tool-` (hyphens instead of underscores) for consistency?

Note this is currently out of sync with the get_tool_names spelling. Perhaps `get_tool_names` should return `list(_the_tools.keys())`?